Dockerfile-debian.template

```diff
@@ -18,6 +18,7 @@ RUN set -eux; \
 		\
 # https://github.com/docker-library/redmine/issues/132
 # (without "gsfonts" we get "Magick::ImageMagickError (non-conforming drawing primitive definition `text' @ error/draw.c/DrawImage/3265):")
```

Comment:
I think we need to adjust this comment somehow (since it's not completely accurate anymore with the addition of `ghostscript`).

Maybe something like this?

```dockerfile
# we need "gsfonts" for generating PNGs of Gantt charts and "ghostscript" for generating PDFs
```